k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -1768,7 +1769,7 @@ public void run() {
                                 final LocalMessage localMessage = localFolder.storeSmallMessage(message, progress::incrementAndGet);
                                 boolean shouldRemoveId = me instanceof CrashedWhileDecryptingException;
                                 updateStatus(account, folder, localFolder, progress, newMessages, todo,
-                                        localMessage, message, shouldRemoveId, messagesToNotify, storageEditor);
+                                        localMessage, message, message.getMessageId(), shouldRemoveId, messagesToNotify, storageEditor);
```

Comment:
so this call is the same as before

Reply:
yes, same call, only getting the messageId. The case that could be more error-prone is when we need to move a message to Suspicious folder, and for some reason the moved message is null (mismatch between uids in the remote and local Suspicious folders). Situations like this one may cause a null pointer exception so the change is a safeguard for that, more or less.